gvr-360video/app/src/main/java/org/gearvrf/gvr360video/Minimal360Video.java

```diff
@@ -33,7 +33,7 @@
     /** Called when the activity is first created. */
     @Override
     public void onInit(GVRContext gvrContext) {
-        GVRScene scene = gvrContext.getNextMainScene();
+        GVRScene scene = gvrContext.getMainScene();
```

Comment:
I agree that we should remove the ability to change the scene. You can still clear it and add different objects if you want to replace it so we are not actually removing functionality.

I think we could implement setMainScene by just copying the relevant properties from the input scene to the one and only main scene. Maybe we could rename it updateMainScene so it is clearer that we are not replacing it, but just changing it.